src/govuk/settings/_links.scss

```
@@ -43,7 +43,7 @@ $govuk-link-underline-thickness: unquote("max(1px, .0625rem)") !default;
 /// @type Number
 /// @access public
 
-$govuk-link-underline-offset: .1em !default;
+$govuk-link-underline-offset: .1578em !default;
```

Shall we update the test in [**src/govuk/helpers/links.test.js**#L63](https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/helpers/links.test.js#L63)?

Probably want to follow the comment block below and explain this is 3px at font size 19

Lots of other magic numbers in this file but could show the calculation too? Not a blocker

```suggestion
$govuk-link-underline-offset: 3px / 19px * 1em !default;
```

---

I don't think the test needs updating. It seems to be testing that changes to the variable are being bubbled through to the related mixin, rather than testing what the value specifically is. 

I'm unsure of using an equation in place of a 'magic' number for a few reasons: 

- There is zero visual difference between how Chromium, WebKit or Gecko renders 0.1578em compared to 0.1578947368em—the [actual value](https://www.w3.org/TR/css-cascade-4/#actual) (sorta the 'normalised' computed value) seems to always be 3px. This remains the case even at high zoom levels. 
- As this equation doesn't evaluate to a round number, we'll end up having slightly different values depending on whether the Sass is compiled by libsass or Dart Sass, due to each compiler having different rounding precision. This could lead to incremental differences that become exacerbated in some circumstances (like #2199). Though admittedly this is unlikely due to the first bullet point, it feels like a complicating factor we could avoid. 
- The use of an equation, and the extra precision it results in, unnecessarily adds to compile times and compiled code weight (albeit, very minorly). 

Ultimately, to my mind, using an equation that always evaluates to the same result doesn't provide any tangible benefit over directly using the result, so I'd prefer to stick with what we have.

---

Absolutely fine, it's not a blocker

You alright adding a comment to explain this is 3px at font size 19 for our future selves?